services/py-api/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM python:3.12.6-bullseye AS builder
+LABEL authors="ivanobreshkov"
+
+# https://medium.com/@albertazzir/blazing-fast-python-docker-builds-with-poetry-a78a66f5aed0
+
+RUN pip install poetry
+
+ENV POETRY_NO_INTERACTION=1 \
+    POETRY_VIRTUALENVS_IN_PROJECT=1 \
+    POETRY_VIRTUALENVS_CREATE=1 \
+    POETRY_CACHE_DIR=/tmp/poetry_cache
+
+WORKDIR /app
+
+COPY pyproject.toml poetry.lock README.md ./
+
+RUN poetry install --no-root && rm -rf $POETRY_CACHE_DIR
```

Comment:
Do you have dev dependencies? This will install everything.

Reply:
Yeah we do. Good catch, I forgot to exclude them.